src/matching-strategy/setup/setupFunctions.js

```diff
@@ -0,0 +1,31 @@
+async function populateIndex (uid, dataset, client) {
+  return await client.index(uid).addDocuments(dataset)
+}
+
+async function sleep (ms) {
+  return await new Promise((resolve) => setTimeout(resolve, ms))
+}
+
+async function watchTasks (client, uid) {
+  let allProcessed = false
+  console.log(`Start update watch for ${uid}`)
+  console.log('-------------')
+  while (!allProcessed) {
```

Comment:
Sorry I wasn't clear enough in my previous message 🙈 
You can also remove the `allProcessed` variable and the `while` loop 🤩